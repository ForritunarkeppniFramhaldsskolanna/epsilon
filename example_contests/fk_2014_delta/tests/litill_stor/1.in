Hello. My name is John. YAY!
