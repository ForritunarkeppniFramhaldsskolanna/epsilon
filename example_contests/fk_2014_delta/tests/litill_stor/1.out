hELLO. mY NAME IS jOHN. yay!
